pkg/apis/app/v1alpha1/types.go

```diff
@@ -20,7 +20,10 @@ Collection of common types among CRDs
 
 // Image is a definition of a Docker image
 type Image struct {
-	Domain    string `json:"domain,omitempty"`
+	Domain string `json:"domain,omitempty"`
+	// +optional
+	// A flag used to indicate the specified image registry is insecure. Defaults to 'false'.
+	Insecure  bool   `json:"insecure,omitempty"`
```

Comment:
I have to give more thoughts on this. Not sure if this would be the correct place to add this flag. I'll get back to it once I finish the work on #368

Reply:
IMHO image and imagestream are different things.
Image in this case defines the fields of a Docker image, not a imagestream. Probably create a new struct would be better, or refactor this Image struct

Reply:
Yeah, @spolti. This is not an attribute for the image, thus to the registry. A `registry` structure or just a flag `insecureRegistry` or something like this would make more sense. Avoid using OpenShift related names in the interface since it will serve both clusters.

Reply:
what about here? 
https://github.com/kiegroup/kogito-cloud-operator/pull/368/files#diff-4007d5c40c32998beb650053c8cb3045R53

:)

Reply:
oh `ImageHandler` is an internal structure to handle image naming/resolution. We need to figure out a structure in the `KogitoService` interface. :cry:

Reply:
Why not then having just a `insecureRegistry` as part of the KogitoServiceSpec ?

Reply:
^ that was my suggestion up there. But it's out of context being in the root of the structure... I'm just nitpicking actually, we can go in this route